docs/man/docker-events.1.md

```diff
@@ -37,6 +37,10 @@ and Docker images will report:
 **--until**=""
    Stream events until this timestamp
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
```

Comment:
How about now @moxiegirl? I tried to apply the style and wording over there here as well.